native/SampleApps/SmartSyncExplorer/src/com/salesforce/samples/smartsyncexplorer/loaders/ContactListLoader.java

```diff
@@ -88,4 +111,55 @@ public ContactListLoader(Context context, UserAccount account) {
 		}
 		return contacts;
 	}
+
+	/**
+	 * Pushes local changes up to the server.
+	 */
+	public synchronized void syncUp() {
```

Comment:
I've moved these methods from ```MainActivity``` here, since they're better suited to the ```Loader``` (not main thread). This enables us to trigger sync operations from a background ```Service```, since loaders are not on UI thread.